ee/fe/modules/300-prometheus/templates/prometheus/pp/prometheus.yaml

```diff
@@ -0,0 +1,256 @@
+  additionalArgs:
+    - name: scrape.timestamp-tolerance
+      value: 10ms
+  initContainers:
+  - command:
+    - sh
+    - -c
+    - chown -vfR 64535:64535 /prometheus-db/
+    image: {{ include "helm_lib_module_common_image" (list . "alpine") }}
+    imagePullPolicy: IfNotPresent
+    name: fix-permissions
+    securityContext:
+      runAsNonRoot: false
+      runAsUser: 0
+    volumeMounts:
+    - mountPath: /prometheus-db
+      name: prometheus-pp-db
+      subPath: prometheus-db
+  containers:
+  - name: prometheus
+    startupProbe:
+      failureThreshold: 300
+    envFrom:
+      - configMapRef:
+          name: prometheus-pp-envs
+  - name: kube-rbac-proxy
+    {{- include "helm_lib_module_container_security_context_read_only_root_filesystem" . | nindent 4 }}
+    image: {{ include "helm_lib_module_common_image" (list . "kubeRbacProxy") }}
+    args:
+    - "--secure-listen-address=$(KUBE_RBAC_PROXY_LISTEN_ADDRESS):9090"
+    - "--client-ca-file=/etc/kube-rbac-proxy/ca.crt"
+    - "--v=2"
+    - "--logtostderr=true"
+    - "--stale-cache-interval=1h30m"
+    ports:
+    - containerPort: 9090
+      name: https
+    env:
+    - name: KUBE_RBAC_PROXY_LISTEN_ADDRESS
+      valueFrom:
+        fieldRef:
+          fieldPath: status.podIP
+    - name: KUBE_RBAC_PROXY_CONFIG
+      value: |
+        upstreams:
+        - upstream: http://127.0.0.1:9090/
+          path: /
+          authorization:
+            resourceAttributes:
+              namespace: d8-monitoring
+              apiGroup: monitoring.coreos.com
+              apiVersion: v1
+              resource: prometheuses
+              subresource: http
+              name: main
+    resources:
+      requests:
+        {{- include "helm_lib_module_ephemeral_storage_only_logs" . | nindent 8 }}
+{{- if not ( .Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
+        {{- include "helm_lib_container_kube_rbac_proxy_resources" . | nindent 8 }}
+{{- end }}
+    volumeMounts:
+    - name: kube-rbac-proxy-ca
+      mountPath: /etc/kube-rbac-proxy
+  - name: config-reloader
+    resources:
+      requests:
+        {{- include "helm_lib_module_ephemeral_storage_logs_with_extra" 20 | nindent 8 }}
+{{- if not ( .Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
+        {{- include "config_reloader_resources" . | nindent 8 }}
+{{- end }}
+  affinity:
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+      - weight: 100
+        podAffinityTerm:
+          labelSelector:
+            matchLabels:
+              app.kubernetes.io/name: prometheus
+              prometheus: longterm
+          topologyKey: kubernetes.io/hostname
+  scrapeInterval: {{ .Values.prometheus.scrapeInterval | default "30s"}}
+  evaluationInterval: {{ .Values.prometheus.scrapeInterval | default "30s" }}
+  externalLabels:
+{{- with .Values.prometheus.externalLabels }}
+    {{- . | toYaml | nindent 4 }}
+{{- end }}
```

Comment:
And here as well. The current main branch does not contain this in the old Prometheus that should be used as a base for the new one.